model-optimizer/extensions/middle/UnsqueezeTileReshapeBlockToInterpolate.py

```diff
@@ -91,19 +94,20 @@ def replace_pattern(self, graph: Graph, match: dict):
         if len(input_shape_of_unsqueeze) not in {4, 5}:
             return
 
-        scale = int64_array([second_input_of_tile.value[d_idx]])
+        scale = np.array([second_input_of_tile.value[d_idx]], dtype=np.float32)
```

Comment:
Is it important to use float32 here, can we use float_array here (it has float64 inside)? 
By the way, if using flaot32 is principal, possibly we need to add function float32_array to simplify code.

Reply:
We use `float32`, because `Interpolate-4` (in nGraph and in plugins) doesn't support `float64` inputs.

Reply:
Added function `float32_array`.